src/collections/_documentation/data-management/rollups.md

```diff
@@ -47,17 +47,30 @@ If everything else fails, grouping falls back to messages. Hereby ideally the gr
 
 ## Customize Grouping with Fingerprints {#custom-grouping}
 
-For some very advanced use cases SDKs can override the Sentry default grouping.
+For some very advanced use cases you can override the Sentry default grouping using the `fingerprint` attribute. In supported SDKs, this attribute can be passed with the event information, and should be an array of strings. 
 
-Two common cases generally come up here:
+If you simply wish to append information, thus making the grouping slightly less aggressive, you can do that as well by adding the special string `{% raw %}{{ default }}{% endraw %}` as one of the items.
```

Comment:
Remove "simply." One of the new best practices for docs is to avoid using words like simple/simply and easy/easily.